content/en/docs/ops/ambient/migrate/index.md

```diff
@@ -0,0 +1,333 @@
+    ✔ Istiod installed
+    ✔ Installation complete
+    {{< /text >}}
+
+1.  Add a namespace label to instruct Istio to automatically inject Envoy
+    sidecar proxies when you deploy your application later:
+
+    {{< text bash >}}
+    $ kubectl label namespace default istio-injection=enabled
+    namespace/default labeled
+    {{< /text >}}
+
+## Deploy the sample application {#bookinfo}
+
+1.  Deploy the [`Bookinfo` sample application](https://istio.io/latest/docs/examples/bookinfo/):
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@
+    service/details created
+    serviceaccount/bookinfo-details created
+    deployment.apps/details-v1 created
+    service/ratings created
+    serviceaccount/bookinfo-ratings created
+    deployment.apps/ratings-v1 created
+    service/reviews created
+    serviceaccount/bookinfo-reviews created
+    deployment.apps/reviews-v1 created
+    deployment.apps/reviews-v2 created
+    deployment.apps/reviews-v3 created
+    service/productpage created
+    serviceaccount/bookinfo-productpage created
+    deployment.apps/productpage-v1 created
+    {{< /text >}}
+
+1.  The application will start. As each pod becomes ready, the Istio sidecar will be
+    deployed along with it.
+
+    {{< text bash >}}
+    $ kubectl get pods
+    NAME                              READY   STATUS    RESTARTS   AGE
+    details-v1-558b8b4b76-2llld       2/2     Running   0          2m41s
+    productpage-v1-6987489c74-lpkgl   2/2     Running   0          2m40s
+    ratings-v1-7dc98c7588-vzftc       2/2     Running   0          2m41s
+    reviews-v1-7f99cc4496-gdxfn       2/2     Running   0          2m41s
+    reviews-v2-7d79d5bd5d-8zzqd       2/2     Running   0          2m41s
+    reviews-v3-7dbcdcbc56-m8dph       2/2     Running   0          2m41s
+    {{< /text >}}
+
+1.  Verify everything is working correctly up to this point. Run this command to
+    see if the app is running inside the cluster and serving HTML pages by
+    checking for the page title in the response:
+
+    {{< text bash >}}
+    $ kubectl exec "$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')" -c ratings -- curl -sS productpage:9080/productpage | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+    {{< /text >}}
+
+## Open the application to outside traffic {#ip}
+
+The Bookinfo application is deployed but not accessible from the outside. To make it accessible,
+you need to create an
+[Istio Ingress Gateway](/docs/concepts/traffic-management/#gateways), which maps a path to a
+route at the edge of your mesh.
+
+1.  Associate this application with the Istio gateway:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/bookinfo-gateway.yaml@
+    gateway.networking.istio.io/bookinfo-gateway created
+    virtualservice.networking.istio.io/bookinfo created
+    {{< /text >}}
+
+### Determining the ingress IP and ports
+
```

Comment:
I'm not sure if this works for me. I believe I needed to install metallb in the past, but maybe I am wrong. Need to investigate this some more. Maybe we can simply use the curls as is done after migrating to ambient mode.